.yeoman/templates/components/src/index.test.jsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import { render } from '@testing-library/react';
-import <%= componentName %> from './index';
+import <%= packageName %> from './index';
 
-describe('<%= componentName %>', () => {
-  shouldMatchSnapshot('should render correctly', <<%= componentName %> />);
+describe('<%= packageName %>', () => {
```

Comment:
This places the component in storybook under `Others` category. It would be nice if the created component can be placed in the correct category.
```suggestion
describe('Components|<%= packageName %>', () => {
```

Reply:
I have done this on `index.stories.jsx` for Components and Containers to get the result you are after: https://github.com/bbc/psammead/pull/2646/commits/5932708d229f0901a4f505e2a8d88115a5be3307

I don't think it is correct for the unit tests though?